### PR TITLE
Feature/custom headers

### PR DESCRIPTION
**Pull request by kwoding**
This should add the requested custom headers support as described in #13 

@kbirger: could you please see if this works in your scenario?

### CODE REVIEW

File: src/model/options.model.ts [hunk 1]
```diff
@@ -6,5 +6,7 @@ export interface Options {
 
   proxy?: string;
 
+  headers?: string;
```
Comment (kbirger) [inline src/model/options.model.ts]: Shouldn't this be `Record<string, string>` or something?

**Review by kbirger — CHANGES_REQUESTED** [review]

**Comment by kbirger** [comment]
I tried this out after changing the types to match your example. I'm not able to get it to work. I can dig in more later, but it doesn't look like the `headers` are getting sent across.

File: src/model/options.model.ts (hunk 1 above)
Comment (kwoding) [reply src/model/options.model.ts]: Just updated the type, please check again, if it does not work, I need to drive into it at a later moment.

**Comment by kwoding** [comment]
Fixed passing the options, can you try again?

File: src/model/options.model.ts [hunk 2]
```diff
@@ -6,5 +6,7 @@ export interface Options {
 
   proxy?: string;
 
+  headers?: any;
```
Comment (kbirger) [inline src/model/options.model.ts:9]:
```suggestion
  headers?: Record<string, string>;
```

**Review by kbirger — APPROVED** [review]
Yup, that did the trick. Thank you for the super quick help!

**Pull request merged**